coverage_point_calculator/src/hexes.rs

```diff
@@ -5,13 +5,40 @@ use rust_decimal_macros::dec;
 
 use crate::{BoostedHexStatus, RadioType, Result};
 
+/// Breakdown of points for a hex.
+///
+/// Example:
+///   Outdoor Wifi with 1-hex boosted 5x,
+///   Rank 2, Assignment AAA:
+///     SplitPoints {
+///       modeled: 16,
+///       base: 8,
+///       boosted: 32
+///     }
+///
+///   Rank 2 splits modeled points in half.
+///   Boost at 5x adds 32 points ( (8 * 5) - 8 )
+#[derive(Debug, Default, Clone)]
+pub struct HexPoints {
+    /// Default points received for hex
+    ///
+    /// (RadioType, SignalLevel) points
+    pub(crate) modeled: Decimal,
```

Comment:
not sure I understand why we are storing this field, are we using it somewhere that I am missing?

Reply:
This field is helpful to have around because different types of radios can achieve the same points through different means. I've made it private to better show that it is not accessed, and added to the fields comment for the same effect.